stub/src/main/java/io/grpc/stub/annotations/RpcMethod.java

```diff
@@ -23,7 +23,17 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotates a method descriptor method to provide metadata for annotation processing.
+ * {@link RpcMethod} contains a limited subset of information about the RPC to assist
```

Comment:
fyi, the import of `import io.grpc.MethodDescriptor.MethodType;` should become `import io.grpc.MethodDescriptor;`

and on line 65: 

`MethodType` -> `MethodDescriptor.MethodType`

Reply:
Done